data/coalition/coalition missions.txt

```diff
@@ -1403,42 +1403,42 @@ mission "Coalition Folklore: Saryd"
 				goto end
 			label seed
-			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and so you do.`
-			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several light rays shine onto it, and as you look up you can only barely see some spotlights high up on the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings on the top. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
-			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper comes down swiftly, as lightning strikes one of the trees on the boy's way, flame-shaped folds raining down from the top of the stage and sticking to it as it falls, blocking the boy's path, so he turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`
+			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and you do so.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
+			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper comes down swiftly, as lightning strikes one of the trees in the boy's way, flame-shaped folds raining down from the top of the stage and sticking to it as it falls, blocking the boy's path, so he turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`
```

Comment:
It's unclear to me what `sticking to it as it falls` means in this context. What is it sticking to? What is falling? Perhaps also make it clearer that the 'white piece of paper' is supposed to represent lightning, because the transition from `A white piece of paper comes down swiftly, as lightning strikes one of the trees on the boy's way...` seems a bit abrupt to me.

Reply:
The "flame-shaped folds (of paper)" are what falls, falling onto the tree struck by the lightning bolt (said tree is falling due to being struck by lightning), and the flames stick to it signifying it's on fire.
I needed something to introduce the fire to the tree so that it's not like the paper representing fire came out of nowhere, but the only other alternative I thought would be the paper tree spinning to its back, which would be modeled to already look on fire, but that would cause issues with the mental image as it spinning would hit the other layers of paper.

Reply:
Fair enough. The changes you've made make what's going on much clearer.